src/dom.js

```diff
@@ -156,6 +184,15 @@ export function childElement(parent, callback) {
  * @return {?Element}
  */
 export function childElementByAttr(parent, attr, opt_value) {
+  if (scopeSelectorSupported == null) {
+    scopeSelectorSupported = isScopeSelectorSupported();
+  }
+  if (scopeSelectorSupported) {
+    if (opt_value !== undefined) {
+      return parent.querySelector(':scope [' + attr + '="' + opt_value + '"]');
```

Comment:
`opt_value` injection here is not safe. I don't think this would come up in our use cases, but as far as API concerned, perhaps we should be careful?